src/ert/storage/migration/to6.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+from pathlib import Path
+
+import xarray as xr
+
+info = "Combining datasets for responses and parameters"
+
+
+def migrate(path: Path) -> None:
```

Comment:
I tried opening some old storages, but did not get the migrations?

Reply:
Should be fixed now